src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
-<html lang="en">
+
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <title>Calendar</title>
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="page__body">
+    <div class="page__container">
```

Comment:
You didn't need another container - remember: divs are just like imaginary boxes and you could actually just use body and the calendar div for all of that

Reply:
I fact I use this page__container div to ensure the calendar layout in center of screen using display: flex. Let me know if is that bad pratice or can lead issues.

Reply:
Unnecessary div:

As @jv-aquino stated, there's no need for another div here. I understand your point, but you can add display: flex to the body if needed.
There's no real problem in adding this div, but if it is the only child, it's also not needed.